lib/languageFeatures/workspaceSymbol.ts

```diff
@@ -25,5 +29,6 @@ export function handleOnWorkspaceSymbol(params: WorkspaceSymbolParams, projectPa
     }
   }
   const symbolsFiltered = symbols.filter(symbol => symbol.name.toLowerCase().includes(params.query.toLowerCase()));
+  console.log('Workspace Symbol ', Date.now() - start);
```

Comment:
```suggestion
```
Maybe a global logging util would be good which can enable/disable different verbosity levels etc...